.taskcluster.yml

```diff
@@ -30,7 +30,7 @@ tasks:
           git fetch {{ event.head.repo.url }} {{ event.head.repo.branch }}
           && git config advice.detachedHead false
           && git checkout {{event.head.sha}}
-          && ./gradlew --no-daemon clean assemble test detektCheck ktlint lint
+          && ./gradlew --no-daemon clean assemble test detektCheck ktlint lint docs
```

Comment:
Do we need to do that for pushes/pull requests or can we save the time? Or asked differently: Can people break things that make the docs not work?

Reply:
No, they can't break things. I set failonerror=false. I figured though we want a /latestBuild URL which points to the most up-to-date javadoc, but yeah I think release is good enough here. Updating...